src/webui/api/appcontroller.cpp

```diff
@@ -565,6 +566,8 @@ void AppController::setPreferencesAction()
         proxyConf.password = it.value().toString();
     proxyManager->setProxyConfiguration(proxyConf);
 
+    if (hasKey(u"proxy_hostname_lookup"_qs))
+        session->setProxyHostnameLookupEnabled(it.value().toBool());
```

Comment:
Move it below `if (hasKey(u"proxy_torrents_only"_qs))`

Reply:
Done.